app/parser/src/TruckIgnore.ts

```diff
@@ -1,6 +1,6 @@
 import { readFileSync } from "fs"
 import { compile } from "gitignore-parser"
-import { warn } from "./log"
+import { warn } from "./log.server"
```

Comment:
Remix uses the *.server glob to determine if it is a clientside file or a server side file

See https://remix.run/docs/en/v1/pages/gotchas#server-code-in-client-bundles